### PR TITLE
fix(frontend): Contracts from genesis should be shown

### LINKED ISSUE #300: [Bug] Contracts from genesis are not shown
## Description
If you open a contract that was deployed in genesis, the account page won't show any contract information.

## Steps to reproduce
1. Open https://explorer.betanet.near.org/accounts/unknown.test
2. Observe that it doesn't show contract information.
3. Check `NEAR_ENV=betanet near state unknown.test` and observe `code_hash: 'DpywzzTNLvpUExfMg8x6yUPuNzEXjQ22D5MpnpKxQunK'`.

### PR DESCRIPTION
**Pull request by icerove**
fix #300

### CODE REVIEW

File: frontend/src/libraries/explorer-wamp/contracts.ts [hunk 1]
```diff
@@ -26,12 +26,19 @@ export default class ContractsApi extends ExplorerApi {
...
-        };
+        if (contractInfo !== undefined) {
+          return {
+            codeHash,
+            transactionHash: contractInfo.hash,
+            timestamp: contractInfo.block_timestamp,
+            accessKeys: accessKeys.keys,
+          };
+        } else {
+          return {
```
Comment (frol) [inline frontend/src/libraries/explorer-wamp/contracts.ts:37]: Let's mark the function to actually return the `Promise<ContractInfo>` and fix the interface definition as definitely don't match this return value.

File: frontend/src/components/contracts/ContractDetails.tsx [hunk 2]
```diff
@@ -29,15 +29,17 @@ export default class extends React.Component<Props, State> {
...
-          this.setState({
-            codeHash: contractInfo.codeHash,
-            transactionHash: contractInfo.transactionHash,
-            timestamp: contractInfo.timestamp,
-            locked: contractInfo.accessKeys.every(
-              (key: any) =>
-                key["access_key"]["permission"]["FunctionCall"] !== undefined
-            ),
-          });
+          if (contractInfo !== undefined) {
```
Comment (frol) [inline frontend/src/components/contracts/ContractDetails.tsx:32]: I think we can live with a single check here :smile:

Comment (icerove) [reply frontend/src/components/contracts/ContractDetails.tsx:32]: Sorry Frol, I do not understand what do you mean...... give me some hint about this

Comment (frol) [reply frontend/src/components/contracts/ContractDetails.tsx:32]: image

**Pull request merged**